README.md

```diff
@@ -1,272 +1,65 @@
-2. Run `brew cask install mactex`
-3. You can check you did it right by running `latex`
-
-> Note that MacTeX will require at least 4.5GB of hard disk space and bandwidth.
->
-> This is due to the fact that it installs every LaTeX addon package offered by [CTAN](https://ctan.org/). Only a few of these packages are required by Manim.
->
-> If you would like a smaller LaTeX install which only contains the packages needed by Manim alone, consider installing BasicTeX and suitable packages with the instructions below. This will require about 1.5 GB of space and bandwidth.
-
-If you have installed MacTeX and are comfortable with it, do not install BasicTeX.
-
-1. Refer to the section on Brew installation to see how to install `brew`
-2. Run `brew cask install basictex`
-3. Ensure that the TeXLiveManager tool `tlmgr` works, by running `tlmgr --version` in the terminal.
-4. Run `sudo tlmgr install standalone preview doublestroke relsize fundus-calligra wasysym physics dvisvgm.x86_64-darwin dvisvgm rsfs wasy cm-super`
-4. Ensure that LaTeX works by running `latex` in the Terminal.
-
-
-### Installing Manim-Community itself
-
-1. Manim-Community runs on Python 3.6+. If you'd like to just use the library, you can install it from PyPI via pip:
-
-```sh
-pip3 install manimce
-```
-
-2. However, if you'd like to contribute to and/or help develop
-`manim-community`, you can clone this branch to your local device. To do
-this, first make sure you have `git` installed. Then, clone this repo by
-executing either
-
-```sh
-git clone git@github.com:ManimCommunity/manim.git
-```
-
-or
 
-```sh
-git clone https://github.com/ManimCommunity/manim.git
-```
+## Installation
 
-depending on whether you want to use HTTPS or SSH. Finally, after having cloned this repo, run the
-following:
+Manim has a few dependencies that need to be installed before it. Please visit
+the
+[documentation](https://manimce.readthedocs.io/en/latest/installation.html)
+and follow the instructions according to your operating system.
 
-```sh
-python3 -m pip install -r requirements.txt
-```
 
 ## Usage
 
-To run your first example Scenes, you can run the following commands:
-
-### For users:
-
-1. Download the `example_scenes/basic.py` file from [GitHub](https://raw.github.com/ManimCommunity/manim/master/example_scenes/basic.py), or place it manually
-in your current working directory with
-```sh
-wget https://raw.github.com/ManimCommunity/manim/master/example_scenes/basic.py
+Here is an example manim script:
+```python
+class SquareToCircle(Scene):
+    def construct(self):
+        circle = Circle()
+        square = Square()
+        square.flip(RIGHT)
+        square.rotate(-3 * TAU / 8)
+        circle.set_fill(PINK, opacity=0.5)
+
+        self.play(ShowCreation(square))
+        self.play(Transform(square, circle))
+        self.play(FadeOut(square))
 ```
 
-2. Run the following command:
+Save this code in a file called `basic.py`.  Now open your terminal in the
```

Comment:
There is already SquareToCircle in `example_scenes/basic.py`, no ? DO i miss something ?